### ISSUE TITLE
multiple choices required validation does not work

### ISSUE THREAD

**Issue by jigarparekh80** [heartexlabs/label-studio#615]
Below config skips second validation relvance2, This perfectly works if I have some other 2nd tag instad of choices. For example, two required tags per region choices and rating works but if i have two required tags as choices then it only validates first one for mandatory criteria, and if i dont select second still it allows to submit task.

<View>
  <View >
    <Labels name="ner" toName="text">
      <Label value="Person" />
      <Label value="Organization" />
    </Labels>
  </View>

  <View>
    <View >
      <Text name="text" value="$text" />
    </View>

    <View>
      <Choices name="relevance" toName="text" perRegion="true" required="true">
      	<Choice value="Relevant" />
        <Choice value="Non Relevant" />
      </Choices>

      <View visibleWhen="region-selected">
      	<Header value="Your confidence" />
      </View>
      <Choices name="relevance2" toName="text" perRegion="true" required="true">
      	<Choice value="Something" />
        <Choice value="Nothing" />
      </Choices>
    </View>

    <View>
      <Header value="Select span after creation to go next"/>
    </View>
  </View>

</View>

**Comment by makseq** [comment]
@jigarparekh80 Thank you very much for your report, it's definitely a bug :slightly_smiling_face:

**Comment by niklub** [comment]
Hi, @jigarparekh80 !

The issue isn't reproduced in [1.0.1 release branch](https://github.com/heartexlabs/label-studio/pull/681) with the following config
```xml
<View>
  <Choices name="relevance" toName="text" required="true">
    <Choice value="Relevant"/>
    <Choice value="Non Relevant"/>
  </Choices>
  <View visibleWhen="region-selected">
    <Header value="Your confidence"/>
  </View>
  <Choices name="relevance2" toName="text" required="true">
    <Choice value="Something"/>
    <Choice value="Nothing"/>
  </Choices>
  <Text name="text" value="$text"/>
  <View>
    <Header value="Select span after creation to go next"/>
  </View>
</View>
```
(I can't actually use exactly yours since it doesn't contain an object tag and enclosing </View>)
We will close this issue as soon as 1.0.1 version will be released. If you continue to face the issue, please feel free to reopen it, providing an exact XML config

**Comment by niklub** [comment]
Hey, this problem is supposed to be fixed following `label-studio>=1.0.1`. Please feel free to reopen the issue in case it happens again.
